server.js

```diff
@@ -14,16 +16,124 @@ import cors from "cors";
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
-
+const listEndpoints = require('express-list-endpoints')
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json(listEndpoints(app))
+//   const movies = data
+//   if (movies) {
+//   res.status(200).json({
+//     success: true,
+//     message: "OK",
+//     body: {
+//       data: movies
+//     }
+//   })
+// } else {
+//   res.status(500).json({
+//     success: false,
+//     message: "Something went wrong",
+//     body: {}
+//   })
+// }
 });
 
+app.get('/movies', (req, res) => {
+  res.json(data)
```

Comment:
is there any specific reason to why you don't have a status and error message for this response?